kubernetes/chart/zulip/templates/NOTES.txt

```diff
@@ -0,0 +1,15 @@
+1. Zulip should now be available on:
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.hosts }}
+  http://{{ .host }}
+{{- end }}
+{{- else }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ template "zulip.name" . }}" -o jsonpath="{.items[0].metadata.name}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
+  kubectl port-forward $POD_NAME 8080:80
+{{- end }}
+
+2. To create a realm so you can sign in:
+
+  kubectl -n {{ .Release.Namespace }} exec -it $(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ template "zulip.name" . }}" -o jsonpath="{.items[0].metadata.name}") -c zulip -- sudo -u zulip /home/zulip/deployments/current/manage.py generate_realm_creation_link
```

Comment:
Can't we use the `$POD_NAME` we just defined?
```suggestion
  kubectl -n {{ .Release.Namespace }} exec -it "$POD_NAME" -c zulip -- sudo -u zulip /home/zulip/deployments/current/manage.py generate_realm_creation_link
```

Reply:
Well, yes and no. I moved POD_NAME outside of the `if` so we can, although the people using this would have to copy-paste the commands into the terminal for them to work. I think it's clear enough now, though.